<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JsonSettingsDialog</class>
 <widget class="QDialog" name="JsonSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>381</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Default Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="vlayoutMain">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Default Startup Values</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelForegroundWindowTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Foreground Window Title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leditForegroundWindowTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Name Of Foreground Window, e.g. Skyrim Special Edition</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelProcessImageName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Process Image Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leditProcessImageName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Name Of Process Image, e.g. TESV.exe, SkyrimSE.exe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelKeyboardShortcut">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Keyboard Shortcut </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlayoutKeyboardShortcut">
        <item>
         <widget class="QLineEdit" name="leditKeyboardShortcut">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Virtual Key ID (VKID)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelActivationMethod">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Activation Method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxActivationMethod">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Nothing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Keyboard Shortcut</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Process Running</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Foreground Window Title</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelStylesheetPath">
        <property name="text">
         <string>Stylesheet Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leditStylesheetPath">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Stylesheet Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelInitialMuteState">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Initial Mute State</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbMuted">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Muted</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlayoutButtons">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="btnSaveDefaults">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Save Defaults</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
